runtime/onert/core/include/backend/IConstantInitializer.h

```diff
@@ -273,6 +273,20 @@ class IConstantInitializer : public ir::OperationVisitor
     }
   }
 
+public:
+  void registerCustomInitializer(const ir::OperandIndex &index, const ir::Operand &obj,
```

Comment:
When does this get used?

Reply:
It's deleted,  cpu's ConstantInitialzer registers it's own custom Initializer-which does tiling while ConstInit- for BCQ.

Reply:
Then can we just implement this in the internal repo? unless it must be here.